rocAL/rocAL/source/meta_data/meta_node_crop.cpp

```diff
@@ -28,7 +28,7 @@ void CropMetaNode::initialize()
     _x1_val.resize(_batch_size);
     _y1_val.resize(_batch_size);
 }
-void CropMetaNode::update_parameters(MetaDataBatch* input_meta_data)
+void CropMetaNode::update_parameters(MetaDataBatch* input_meta_data, bool segmentation)
```

Comment:
Add const to bool segmentation if we dont modify this variable

Reply:
Resolved in the latest commit